### ISSUE TITLE
Undeclared dependency Test::Exception

### ISSUE THREAD

**Issue by eserte** [singingfish/p5-array-circular#3]
t/03-bugs.t fails if Test::Exception is not installed:
```
Can't locate Test/Exception.pm in @INC (@INC contains: /home/cpansand/.cpan/build/2019050903/Array-Circular-0.004-qBdhVZ/blib/lib /home/cpansand/.cpan/build/2019050903/Array-Circular-0.004-qBdhVZ/blib/arch /usr/local/lib64/perl5 /usr/local/share/perl5 /usr/lib64/perl5/vendor_perl /usr/share/perl5/vendor_perl /usr/lib64/perl5 /usr/share/perl5 .) at t/03-bugs.t line 6.
BEGIN failed--compilation aborted at t/03-bugs.t line 6.
t/03-bugs.t .............. 
Dubious, test returned 2 (wstat 512, 0x200)
No subtests run 
```

**Comment by eserte** [comment]
Happens also with 0.005.

**Comment by dr-kd** [comment]
oh dear sorry I missed this.  I usually work on this on the train when I don't have internet access is my excuse.  Fixed by 883834b5405
